package.json

```diff
@@ -162,7 +162,8 @@
 	"prettier": {
 		"printWidth": 100,
 		"useTabs": true,
-		"singleQuote": true
+		"singleQuote": true,
+		"endOfLine": "auto"
```

Comment:
Because of windows-trouble

Reply:
Isn't it better to just setup a `.gitattributes` file? I think using Linux endings is better, and Prettier offers a step-by-step on how to do that: https://prettier.io/docs/en/options.html#end-of-line

Reply:
I've added a `.gitattributes` file to the master branch, so I don't think this is needed anymore.

Reply:
I have a feeling that this is either already wrong, or it doesn't work, I get this on master even after your `.gitattributes` on windows:
```bash
$ npm run format-check

> universal-trakt-scrobbler@1.0.0 format-check 
> npm run prettier -- --check


> universal-trakt-scrobbler@1.0.0 prettier 
> prettier "**/*.{js,jsx,ts,tsx}" "--check"

Checking formatting...
[warn] src\api\TraktSearch.ts
[warn] src\common\BrowserStorage.tsx
[warn] src\common\Events.ts
[warn] src\common\Requests.ts
[warn] src\global.d.ts
[warn] src\modules\history\history.tsx
[warn] src\modules\options\components\OptionsListItem.tsx
[warn] src\modules\options\options.tsx
[warn] src\modules\options\OptionsApp.tsx
[warn] src\modules\popup\popup.tsx
[warn] src\modules\popup\PopupApp.tsx
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
```

Reply:
You have to re-clone your local clone for it to work.